spec/coffee/providers/google-map-api-provider.spec.coffee

```diff
@@ -22,3 +22,37 @@ describe 'uiGmapGoogleMapApiProvider', ->
     mapScriptLoader.load(options)
     lastScriptIndex = document.getElementsByTagName('script').length - 1
     expect(document.getElementsByTagName('script')[lastScriptIndex].src).toContain('http://maps.google.cn/maps/api/js')
+
+  describe 'on Cordova devices', ->
+    beforeAll ->
+      window.navigator.connection = {}
+      window.Connection =
+        WIFI: 'wifi'
+        NONE: 'none'
+
+    afterAll -> delete window.navigator.connection
+
+    it 'should include the script when the device is online', ->
+      window.navigator.connection.type = window.Connection.WIFI
+
+      options = { v: '3.17', libraries: '', language: 'en', sensor: 'false', device: 'online' }
+      mapScriptLoader.load(options)
+
+      lastScriptIndex = document.getElementsByTagName('script').length - 1
+      expect(document.getElementsByTagName('script')[lastScriptIndex].src).toContain('device=online')
```

Comment:
I tried to remove scripts that match `document.querySelectorAll(script[id^=ui_gmap_map_load_])` after every tests but I has issue with that.
The `device=online` stuff is a little hack to check for the right script, since we include all the options in the script url.